src/pages/HomePage/HomePage.jsx

```diff
@@ -10,25 +10,28 @@ function HomePage() {
   const [inputValue, setInputValue] = useState('');
   const history = useHistory();
 
-  const getAllCitiesByName = async event => {
-    setInputValue(event.target.value.toLowerCase());
-
+  useEffect(async () => {
```

Comment:
In previous comments you ask me is I really need the effect. Actually I need this effect, because If I made the request to API from changeValueHandler (getAllCitiesByName) a bug appears. The request to API is made with previous value of inputValue. For example if user write 'Sofia' the request is made with 'Sofi'.